app/controllers/api/invitations_controller.rb

```diff
@@ -28,8 +40,4 @@ def invitation_parser
     @invitation_parser ||= InvitationParser.new(@invitations)
   end
 
-  def resource_serializer
```

Comment:
@gdpelican would be good if we can use normal invitation models in the existing invitation system. Is there anything stopping us from going that way?